core/templates/pages/learner-dashboard-page/modal-templates/remove-activity-from-learner-dashboard-modal.template.html

```diff
@@ -7,8 +7,8 @@ <h3 translate="I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_HEADER" translate-va
 </div>
 
 <div class="modal-footer">
-  <button class="btn btn-secondary" ng-click="cancel()" translate="I18N_MODAL_CANCEL_BUTTON"></button>
+  <button class="btn btn-secondary" (click)="cancel()" translate="I18N_MODAL_CANCEL_BUTTON"></button>
   <button class="btn btn-danger protractor-test-confirm-delete-interaction"
-          ng-click="remove()" translate="I18N_LEARNER_DASHBOARD_REMOVE_BUTTON">
+          (click)="remove()" translate="I18N_LEARNER_DASHBOARD_REMOVE_BUTTON">
```

Comment:
Do these translate tags need to be migrated properly?

Reply:
I have updated the code.